hw1/02-palindrome.html

```diff
@@ -9,17 +9,18 @@
       href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
     <script defer src="02-palindrome.js"></script>
   </head>
-  <body>
-    <main>
-      <section>
-        <h1>Exercise 02 - Palindrome</h1>
-        <div>
-          <label>Enter a positive number:</label>
-          <input type="number" />
+  <body class="bg-primary">
+    <main class="container-fluid">
```

Comment:
Similar to what mnxn said, I think the container-fluid class here can be changed to be a smaller width or include a different element within that can be used to set maximum width. You could also limit width of all elements within container if you don't want to target the container itself.